CoffeeChat/Controller/EditingViewController.swift

```diff
@@ -82,36 +82,61 @@ class EditingViewController: UIViewController {
     private let tableView = UITableView(frame: .zero, style: .grouped)
 
     // MARK: - Display Settings
```

Comment:
I notice some sections begin with `MARK` and some don't. This make sense but I'm pointing it out just in case.